Gemfile

```diff
@@ -58,18 +55,18 @@ gem 'sprockets-rails', require: 'sprockets/railtie'
 gem 'statsd-instrument'
 gem 'twitter-text'
 gem 'tzinfo-data'
+gem 'webpacker', github: 'rails/webpacker'
```

Comment:
Okay. Just did what the webpacker readme recommended. Will gladly pin the version.